sdk/tests/ContractIdTest.cc

```diff
@@ -0,0 +1,97 @@
+/*-
+ *
+ * Hedera C++ SDK
+ *
+ * Copyright (C) 2020 - 2022 Hedera Hashgraph, LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License")
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+#include "ContractId.h"
+
+#include <gtest/gtest.h>
+#include <proto/basic_types.pb.h>
+
+using namespace Hedera;
+
+class ContractIdTest : public ::testing::Test
+{
+protected:
+  [[nodiscard]] inline const uint64_t& getTestNum() const { return mTestNum; }
+
+private:
+  const uint64_t mTestNum = 10;
```

Comment:
I think it would make sense to have a different test number each for shard, realm, and contract. In the current tests, getting wires crossed (e.g. setting shard to desired realm) might be missed

Reply:
Sure thing, done